portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Use the interface instead of actual object (i.e. List<String> comments = new ArrayList..

Reply:
In order to do that I have to also import "List" (in addition to ArrayList and Arrays). 
Do you think it's better to do it this way?

Reply:
No problem in doing that, the main point here is that it's always preferable to use the interface rather than the actual implementing object (this applies both for class fields, method arguments, inner parameters, etc.).

Reply:
alright. thanks